docs/plugins/verifier/notation.md

```diff
@@ -64,7 +35,8 @@ spec:
           signatureVerification:
             level: strict
           trustStores:
-            - ca:certs
+            - ca:ca-certs
```

Comment:
we could add a comment clarifying the trustStore must be `trust-store-type:trust name` specified in `verificationCertStores`.